diode-server/gen/netbox/server_utils.go

```diff
@@ -554,46 +554,42 @@ func GetObjectTypeName(objectType string) (string, error) {
 
 func GetPrimaryValue(entity *pb.Entity) (string, error) {
     switch e := entity.GetEntity().(type) {
-    case *pb.Entity_AsnRange:
-        return fmt.Sprintf("%v", e.AsnRange.Name), nil
     case *pb.Entity_Asn:
         return fmt.Sprintf("%v", e.Asn.Asn), nil
-    case *pb.Entity_CircuitGroup:
-        return fmt.Sprintf("%v", e.CircuitGroup.Name), nil
+    case *pb.Entity_AsnRange:
```

Comment:
I changed this to a stable ordering (I think), this is mostly just shuffling into alphabetical.